src/azul/service/manifest_service.py

```diff
@@ -815,15 +824,39 @@ def manifest_config(self) -> ManifestConfig:
                               combine_script='return new ArrayList(params._agg.fields)',
                               reduce_script=reduce_script)
         es_search = es_search.extra(size=0)
-        response = es_search.execute()
-        assert len(response.hits) == 0
+        try:
+            response = es_search.execute()
+        # If timeout is exceeded the request is partitioned to reduce load on ES
+        except ConnectionTimeout:
+            fields = self._partition_search(es_search)
+        else:
+            assert len(response.hits) == 0
+            fields = response.aggregations.fields.value
         return {
             'contents': {
                 value: value.split('.')[-1]
-                for value in sorted(response.aggregations.fields.value)
+                for value in sorted(fields)
             }
         }
 
+    def _partition_search(self, es_search: Search) -> Set[str]:
+        requests = [es_search.query(Q('bool',
+                                      must=Q('prefix',
+                                             **{'bundles.uuid.keyword': p})))
+                    for p in map(''.join, product('0123456789abcdef'))]
```

Comment:
why not list('0123456789abcdef') rather than this mapping?

Reply:
Good catch, I was using [this bit](https://github.com/DataBiosphere/azul/blob/bee1c63207d581b06709898487f75f8051669d9c/src/azul/azulclient.py#L205) as reference, but your suggestion is more concise.